python/instrumentation/openinference-instrumentation-mistralai/src/openinference/instrumentation/mistralai/_response_attributes_extractor.py

```diff
@@ -25,34 +27,34 @@
 class _ResponseAttributesExtractor:
     def get_attributes_from_response(
         self,
-        response: "ChatCompletionResponse",
+        response: Any,
```

Comment:
Why any?

Reply:
This method accepts different types that have the same shape (streaming vs. non-streaming cases).